pushapkscript/Dockerfile.test

```diff
@@ -4,10 +4,15 @@ FROM python:${PYTHON_VERSION}
 
 WORKDIR /app
 
+RUN apt-get update && \
+    apt-get install -y default-jdk && \
+    apt-get clean && \
+    curl -L https://github.com/google/bundletool/releases/download/1.15.4/bundletool-all-1.15.4.jar -o /app/bundletool.jar
+
+ENV BUNDLETOOL_PATH=/app/bundletool.jar
+
 COPY MANIFEST.in setup.py tox.ini /app/
 COPY requirements/ /app/requirements/
-COPY docker.d/image_setup.sh /app/
```

Comment:
Not related to the AAB efforts...I just noticed that image_setup.sh is obsolete.